src/content/docs/guides/Flutter (Mobile)/index.md

```diff
@@ -0,0 +1,245 @@
+
+Inne edytory to śmierć, ale to już wasza sprawa.
+
+Poniżej, po instalacji Fluttera, załączam listę wtyczek i ustawień do VSCode, które ułatwią/umożliwią Wam pracę.
+
+## Instalacja Fluttera
+
+Dla odmiany od np. technologii zbudowanych na JSie, Flutter ma dość jednolite środowisko. Oczywiście czasami problemem może być jego wieloplatformowość i to że pod spodem musi zależeć trochę od natywnego Androida i iOSa (no i przegladarkowych i desktopowych) warstw. Jednak sama flutterowa warstwa jest dość jednolita i nie ma np. 15 różnych środowisk uruchomieniowych, kilku standardów, sposobów rozwiązywania ścieżek itp. Tutaj środowisko jest jedne, którego kierunek rozwoju jest dość silnie wytyczane przez Flutter and Dart team w Google (_z małą gwiazdką o Flocku_). A sposoby na instalacje są właściwie tylko dwa:
+
+### Instalacja bezpośrednia 1:1 z dokumentacją
+
+Wszystko jest dokładnie opisane w docsach: [Instrukcja instalacji Fluttera](https://flutter.dev/docs/get-started/install). Ale w skrócie potrzebujemy:
+
+- Fluttera i Darta
+- Android Toolchain / Android Studio
+- iOS Platform / XCode (tylko na Macu)
+- (opcjonalnie) Symulatory iOS i/lub Androida
+
+### Instalacja przez [FVM (Flutter Version Manager)](https://fvm.app/) - polecane
+
+Wadą pierwszej metody jest to, że Flutter jest zainstalowany globalnie i jeżeli pracujemy nad kilkoma projektami, to musimy używać jednej wersji Fluttera. FVM pozwala na zainstalowanie wielu wersji Fluttera i łatwe przełączanie między nimi.
+
+Dlatego moim zaleciem jest zainstalowanie FVM i korzystanie z niego od samego początku: https://fvm.app/documentation/getting-started/installation
+Jeśli zdecydujecie się na FVM, to Fluttera i Darta nie musicie instalować globalnie, bo FVM zrobi to za was.
+Nadal musicie zainstalować Android Studio i XCode, zgodnie z instrukcją z dokumentacji Fluttera.
+
+:::caution
+Nie polecam instalacji zarówno z FVM i globalnej instalacji Fluttera, bo to może tam to się gryźć. Jak chcecie się przenieść z globalnej instalacji na FVM, to polecam odinstalować globalnego Fluttera.
+:::
+
+:::note
+Jeśli korzystacie z FVM, trzeba proxować wszystkie komendy przez dodanie przedrostka `fvm`:
+
+```shell
+fvm flutter doctor
+
+fvm flutter {command}
+
+fvm dart {command}
+```
+
+:::tip
+Można też ustawić aliasy w terminalu, żeby nie pisać `fvm` za każdym razem. Instrukcja jest w docsach: https://fvm.app/documentation/guides/running-flutter
+:::
+
+:::note
+Jeśli korzystacie z FVM, trzeba zmienić ustawienia Edytorów, żeby automatycznie wykrywały instancję z FVM w tym projekcie. Instukcja dla VSCode'a: https://fvm.app/documentation/guides/vscode
+:::
```

Comment:
bardzo dużo tu notesów, tipów i cautionów (jedno nawet znestowane w drugim 😭), zapisałbym to jako zwykły tekst raczej

Reply:
ale tak są kolorki